### PR TITLE
Bugfix for `paint!`

### PR DESCRIPTION
**Pull request by lmh91**
Regarding #186

### CODE REVIEW

**Comment by fhagemann** [comment]
Can you please post the example how the plots for the file that revealed the bug looks like?

**Comment by fhagemann** [comment]
I ran your code example from #186 and it looks weird (left my file, right the example config file):

after_PR

**Comment by fhagemann** [comment]
The `difference` does not seem to work properly (the inner box is not properly cut out)

after_PR2

**Comment by fhagemann** [comment]
And after refinement, it looks like this:
after_PR3

Are we somehow making sure to not paint the parts of the surfaces that are subtracted?

File: src/PotentialSimulation/Painting/Painting.jl [hunk 1]
```diff
@@ -44,18 +44,30 @@ function paint!(pointtypes, potential, face::AbstractSurfacePrimitive{T}, geomet
     widths_ax1 = diff(get_extended_ticks(grid[1]))
     widths_ax2 = diff(get_extended_ticks(grid[2]))
     widths_ax3 = diff(get_extended_ticks(grid[3]))
+    Δw_max_factor = T(1e-2)
```
Comment (fhagemann) [inline src/PotentialSimulation/Painting/Painting.jl:47]: This is just a scaling factor?
I must admit, I haven’t really looked into Δw_mac ...

Comment (lmh91) [reply src/PotentialSimulation/Painting/Painting.jl:47]: Basically, yes. 
It's a bit tricky to adjust the tolerance in the `in` method for the local grid. 
One parameter, like it is now, is certainly not the perfect solution. 
But it works in most cases.

**Comment by lmh91** [comment]
So now it looks like this:
test

**Review by fhagemann — APPROVED** [review]

**Pull request merged**